src/rcl_bindings.cpp

```diff
@@ -65,11 +68,23 @@ NAN_METHOD(Init) {
   rcl_init_options_t init_options = rcl_get_zero_initialized_init_options();
   rcl_ret_t ret = rcl_init_options_init(&init_options, allocator);
 
+  // preprocess Context
   RclHandle* context_handle = RclHandle::Unwrap<RclHandle>(info[0]->ToObject());
   rcl_context_t* context =
       reinterpret_cast<rcl_context_t*>(context_handle->ptr());
+
+  // preprocess argc & argv
+  v8::Local<v8::Array> jsArgv = v8::Local<v8::Array>::Cast(info[1]);
+  int argc = jsArgv->Length();
+  const char *argv[argc];
```

Comment:
I found the detailed information https://docs.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/compiler-error-c2131?view=vs-2019, I think it's a ms compiler-specific error.

Also, |argv| should not be const as it will be assigned in L82

Reply:
Fixed with 586cd4fa72652fdacf4758f20c88616bed0eabcf